### PR TITLE
add db cli list and delete action for users

### PR DESCRIPTION
**Pull request by flo-mic**
This pull request adds a list and delete functionality to the HiveMind-database CLI. It allows to list all user names and also delete keys based on username and api key.

### CODE REVIEW

File: jarbas_hive_mind/database/__main__.py [hunk 1]
```diff
@@ -17,6 +17,14 @@ def main():
         with ClientDatabase() as db:
             db.add_client(
                 args.name, args.access_key, crypto_key=args.crypto_key)
+    if args.action == 'list':
+        with ClientDatabase() as db:
+            for x in db.get_clients():
+                print (x)
```
Comment (JarbasAl) [inline jarbas_hive_mind/database/__main__.py:23]: extra space after `print` ?

File: jarbas_hive_mind/database/__main__.py [hunk 2]
```diff
@@ -17,6 +17,14 @@ def main():
...
+    if args.action == 'delete':
+        with ClientDatabase() as db:
```
Comment (JarbasAl) [inline jarbas_hive_mind/database/__main__.py:25]: can you add some error checking and LOG if client does not exist?

**Comment by flo-mic** [comment]
I just added a error handler if you try to remove a key that is not present in the db. Also I improved the list function to list id, name and api key but not the crypto key as I believe this should not be visible from a CLI. Shall I also include a crypto key change function?

**Comment by JarbasAl** [comment]
thanks for your contributions, these scripts now live in https://github.com/JarbasHiveMind/hmcli/tree/master/hmcli

A full cli utility for managing hivemind is in the works

**Pull request closed without merging**